qanary_commons/src/main/java/eu/wdaqua/qanary/communications/RestTemplateConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package eu.wdaqua.qanary.communications;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
@dschiese The context is hard to understand and connect with the other code changes. Please add a comment on the purpose and how the configuration is used in the code.